src/Block.tsx

```diff
@@ -0,0 +1,373 @@
+        marginBottom && parseSpacing("marginBottom", marginBottom, SIZES.base),
+        marginLeft && parseSpacing("marginLeft", marginLeft, SIZES.base),
+        marginVertical &&
+          parseSpacing("marginVertical", marginVertical, SIZES.base),
+        marginHorizontal &&
+          parseSpacing("marginHorizontal", marginHorizontal, SIZES.base)
+      ];
+    }
+
+    if (type === "padding") {
+      return [
+        padding && getSpacing(type, padding, SIZES.base),
+        paddingTop && parseSpacing("paddingTop", paddingTop, SIZES.base),
+        paddingRight && parseSpacing("paddingRight", paddingRight, SIZES.base),
+        paddingBottom &&
+          parseSpacing("paddingBottom", paddingBottom, SIZES.base),
+        paddingLeft && parseSpacing("paddingLeft", paddingLeft, SIZES.base),
+        paddingVertical &&
+          parseSpacing("paddingVertical", paddingVertical, SIZES.base),
+        paddingHorizontal &&
+          parseSpacing("paddingHorizontal", paddingHorizontal, SIZES.base)
+      ];
+    }
+  };
+
+  const {
+    width,
+    height,
+    flex,
+    noflex,
+    row,
+    column,
+    center,
+    middle,
+    left,
+    right,
+    top,
+    bottom,
+    card,
+    shadow,
+    elevation,
+    // colors
+    color,
+    primary,
+    secondary,
+    tertiary,
+    black,
+    white,
+    gray,
+    error,
+    warning,
+    success,
+    info,
+    // positioning
+    space,
+    radius,
+    wrap,
+    animated,
+    theme,
+    safe,
+    style,
+    children,
+    scroll
+  } = props;
+
+  const excludeProps = [
+    "margin",
+    "marginTop",
+    "marginRight",
+    "marginBottom",
+    "marginLeft",
+    "marginVertical",
+    "marginHorizontal",
+    "padding",
+    "paddingTop",
+    "paddingRight",
+    "paddingBottom",
+    "paddingLeft",
+    "paddingVertical",
+    "paddingHorizontal"
+  ];
+
+  const extraProps = Object.keys(props).reduce<ExtraProps<BlockProps>>(
+    (prop, key) => {
+      if (!excludeProps.includes(`${key}`)) {
+        prop[key] = props[key];
+      }
+      return prop;
+    },
+    {}
+  );
+
+  const { SIZES, COLORS } = mergeTheme(expoTheme, theme);
+  const marginSpacing = getSpacings("margin");
+  const paddingSpacing = getSpacings("padding");
+
+  const blockStyles = StyleSheet.flatten([
+    styles.block,
+    flex && { flex: flex === true ? 1 : flex },
+    (!flex || noflex) && { flex: 0 },
+    row && styles.row,
+    column && styles.column,
+    center && styles.center,
+    middle && styles.middle,
+    left && styles.left,
+    right && styles.right,
+    top && styles.top,
+    bottom && styles.bottom,
+    marginSpacing,
+    paddingSpacing,
+    wrap && styles.wrap,
+    shadow && {
+      elevation,
+      shadowColor: COLORS.black,
+      shadowOffset: { width: 0, height: elevation - 1 },
+      shadowOpacity: 0.1,
+      shadowRadius: elevation
+    },
+    space && { justifyContent: `space-${space}` },
+    card && { borderRadius: 12 },
```

Comment:
Apparently SIZES.border is not declared, and I don't know what the value should be

Reply:
We need to add a new value for SIZES.borderRadius